docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -696,33 +719,61 @@ REST API asks the Gameroom UI to sign it with Alice's information.
-   is placed in the "pending payloads" queue for unpeered nodes.
+   is placed in the "unpeered payloads" queue for unpeered nodes.
 
 #### I-2.4. Acme node peers with Bubba Bakery node
 
 Before the `​CircuitManagementPayload​` message can be validated, every member of
 the circuit must be connected (peered).
 
 The admin service on the Acme Splinter node (which has the service ID
-`admin::acme-node-000​`) uses a `​PeerConnector​` to request connections with the
-members. The `​PeerConnector​` joins a transport and a network in order to enable
-adding peers at runtime, without having knowledge of the underlying transport.
-
-1. Acme's admin service calls `​PeerConnector.connect​` with the node ID and the
-   endpoint listed in the proposed circuit. If the node is already connected, the
-   peer connector returns "​Ok​". If the node is not connected, the peer connector
-   creates the connection and, if successful, adds the connection to the Splinter
-   network.
-
-2. After the connection has been created, a message exchange starts for peer
+`admin::acme-node-000​`) is able to send and receive messages using a
+`PeerManagerConnector` to request the creation of a peer and to be notified of
+the status of a peering request. The `PeerManagerConnector` sends requests to
+the `PeerManager`, which works on the networking layer to handle the creation
+and authorization of peers. Peering requests are counted by the
+`PeerManagerConnector` to ensure connections remain only as long as they are
+needed.
+
+1. Acme's admin service uses the `PeerManagerConnector.add_peer_ref` with the
+   node ID and the endpoints listed in the proposed circuit to begin the peering
+   process.
+
+2. `PeerManagerConnector.add_peer_ref` sends a message,
+   `PeerManagerRequest::AddPeer` including the `peer_id` and `endpoints`, is
+   sent to the `PeerManager`. A `PeerRef` is returned to the admin service that
+   represents the incremented count for peer requests to ensure the connection
+   is maintained for as long as it is needed. The admin service must hold on to
+   this `PeerRef` for as long as the peer is used.
+
+3. The `PeerManager` receives the `PeerManagerRequest` message and uses
+   `PeerManager.add_peer` to create the connection.
+
+4. After the connection has been created, a message exchange starts for peer
    authorization (described in ​Appendix A​).
 
-3. When peer authorization succeeds, the Acme admin service is notified that the
-   Bubba Bakery node (`​bubba-node-000​`) has been successfully authorized. Acme
-   and Bubba are now peers.
-
-4. The `​CircuitManagmentPayload​` is removed from the "pending payloads" queue
-   and is passed to the admin service handler for pending circuit payloads.
+5. If the connection is authorized, the connection is added to the Splinter
+   network. The `PeerManager` then sends a `Connected` message to its
+   subscribers.
+
+6. The `​CircuitManagmentPayload​` is removed from the `unpeered_payloads` queue
```

Comment:
Mention that this is triggered by the admin service getting the Connected nofication